build/gen_stub.php

```diff
@@ -1015,13 +1015,19 @@ class PropertyInfo
     public $type;
     /** @var Expr|null */
     public $defaultValue;
+    /** @var string|null */
+    public $defaultValueString;
+    /** @var bool */
+    public $isReadonly;
```

Comment:
```suggestion
    public $isDocReadonly;
```
To clarify how this differs from the READONLY flag?

Reply:
Yes, it's a good idea. I applied this change while committing to master.